roottest/root/io/references/CMakeLists.txt

```
@@ -1,7 +1,71 @@
-#-------------------------------------------------------------------------------
-#
-# Placeholder file to translate the tests to the new CTest system. Meanwhile we
-# define a CTest test that calls 'make' in ${CMAKE_CURRENT_SOURCE_DIR}
-#
-#-------------------------------------------------------------------------------
-ROOTTEST_ADD_OLDTEST(LABELS longtest)
+ROOTTEST_ADD_TESTDIRS()
+
+
+ROOTTEST_ADD_TEST(assertManyRefs
+                  MACRO assertManyRefs.C
+                  OUTREF assertManyRefs.ref)
+
+ROOTTEST_COMPILE_MACRO(A.C
+                       FIXTURES_SETUP root-io-references-A-fixture)
+
+ROOTTEST_COMPILE_MACRO(lotsRef.C
+                       ROOTEXE_OPTS -e ".L A.C+"
```

This works and is fine.
The following should also work:
```suggestion
                       BUILDLIB A_C
```
(although it is not actually used yet ... so who knows it works as 'advertised' :) )

---

Yes, seems to be works